tools/ros2_dependencies.repos

```diff
@@ -19,3 +19,7 @@ repositories:
     type: git
     url: https://github.com/ros-perception/vision_opencv.git
     version: ros2
+#  SteveMacenski/slam_toolbox:
```

Comment:
Please remove this - don't leave commented out things around

Reply:
done